src/components/Card/Card.stories.js

```diff
@@ -0,0 +1,251 @@
+import {
+  SbCard,
+  SbCardHeader,
+  SbCardContent,
+  SbCardFooter
+} from '.'
+
+import SbLink from '../Link'
+import SbLoading from '../Loading'
+
+import {
+  SbMenu,
+  SbMenuButton,
+  SbMenuItem,
+  SbMenuList
+} from '../Menu'
+
+const CardTemplate = args => ({
+  components: {
+    SbLink,
+    SbCard,
+    SbCardHeader,
+    SbCardContent,
+    SbCardFooter
+  },
+  props: Object.keys(args),
+  template: `
+    <SbCard>
+      <SbCardHeader :as="as" :title="title" />
+
+      <SbCardContent>
+        <div
+          style="background-color: #f5f5f5;
+          width: 100%;
+          height: 300px;"
+        />
+      </SbCardContent>
+
+      <SbCardFooter>
+        <SbLink
+          href="https://storyblok.com"
+          label="Storyblok Website"
+          icon-right="chevron-right"
+        />
+      </SbCardFooter>
+  </SbCard >
+  `
+})
+
+export default {
+  title: 'SbCard',
+  component: SbCard,
+  parameters: {
+    docs: {
+      description: {
+        component: 'Cards or simple content box, used to divide content elements on page.'
+      }
+    }
+  },
+  args: {
+    as: 'span',
+    title: 'Card Title'
+  },
+  argTypes: {
+    as: {
+      name: 'as',
+      description: 'Define which tag element will be used to title text.',
+      control: {
+        type: 'text'
+      }
+    },
+    title: {
+      name: 'title',
+      description: 'Title for the card.',
+      control: {
+        type: 'text'
+      }
+    }
+  }
+}
+
+export const Default = CardTemplate.bind({})
+
+export const WithoutHeaderAndFooter = args => ({
+  components: {
+    SbCard,
+    SbCardContent
+  },
+
+  props: Object.keys(args),
+
+  template: `
+    <SbCard>
+      <SbCardContent>
+        <div
+          style="background-color: #f5f5f5;
+          width: 100%;
+          height: 50px;"
+        />
+      </SbCardContent>
+    </SbCard >
+  `
+})
+
+WithoutHeaderAndFooter.parameters = {
+  docs: {
+    description: {
+      story: 'Cards without the header and footer are for showing only the content, but continue to divide the content while maintaining the style of cards.'
+    }
+  }
+}
+
+export const WithLoadingComponent = args => ({
+  components: {
+    SbLoading,
+    SbCard,
+    SbCardHeader,
+    SbCardContent,
+    SbCardFooter
+  },
+
+  props: Object.keys(args),
+
+  template: `
+    <SbCard>
+      <SbCardHeader :title="title" />
+
+      <SbCardContent>
+        <div
```

Comment:
Will it be necessary for the dev/user to place this container with these styles in order to centralize the loading? Perhaps using a class in the container when loading is active makes it more automatic.

Reply:
The previous approach places the loading component directly in the component. What do you think? Is is better? Because I believe that, if we will add a class to card component when in loading, I believe that should be good implement the loading directly. The loading component should be placed in absolute position in content. What do you think?

Reply:
Yes, I agree!

Reply:
Great